> ##[error]eng/tools/RepoTasks/RepoTasks.csproj(0,0): error NU1101: Unable to find package Internal.AspNetCore.BuildTasks

@dougbu Should we push this to one of the AzDO feeds?

--- a/NuGet.config
+++ b/NuGet.config
@@ -11,6 +11,7 @@
     <add key="darc-pub-aspnet-EntityFrameworkCore-be2fd3d" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-aspnet-EntityFrameworkCore-be2fd3d4/nuget/v3/index.json" />
     <add key="darc-pub-aspnet-AspNetCore-Tooling-7471a3d" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-aspnet-AspNetCore-Tooling-7471a3da/nuget/v3/index.json" />
     <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
+    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
     <add key="dotnet3" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3/nuget/v3/index.json" />
     <add key="dotnet3-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3-transport/nuget/v3/index.json" />

